CONTRIBUTORS.md

```diff
@@ -2001,11 +1998,6 @@ Noida
 - Bio: IB Diploma high school student.
 - GitHub: [DHDaniel](https://github.com/DHDaniel)
 
-#### Name: [Christian Heinrichs](https://github.com/christianheinrichs)
```

Comment:
Why are you deleting this name from the readme? Please add it back.

Reply:
Fixed, sorry. Don't know how that happened.